### PR TITLE
Propose adding a note for o365 group

### PR DESCRIPTION
**Pull request by get-itips**
https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/issues/559

@MikePlumleyMSFT @AdwoodMicrosoft Please review if adding this information has some value for the article.

Also removed quotes from metadata which is not needed.

Thanks

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/add-a-sharepoint-geo-admin.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ Some services - such as the term store - are administered from the central locat
 
 Global administrators and SharePoint Online administrators continue to have access to settings in the central location and all satellite locations.
 
+>[!Note]
+> Geo Admins cannot manage Office 365 group-connected SharePoint sites but can add themselves as Site Collection Admins to an Office 365 group-connected SharePoint sites using [Set-SPOUser](https://docs.microsoft.com/powershell/module/sharepoint-online/set-spouser?view=sharepoint-ps) PowerShell cmdlet.
```
Comment (JohanFreelancer9) [inline Enterprise/add-a-sharepoint-geo-admin.md]:
```suggestion
> Geo Admins cannot manage Office 365 group-connected SharePoint sites but can add themselves as Site Collection Admins to Office 365 group-connected SharePoint sites using the [Set-SPOUser](https://docs.microsoft.com/powershell/module/sharepoint-online/set-spouser?view=sharepoint-ps) PowerShell cmdlet.
```

File: Enterprise/add-a-sharepoint-geo-admin.md [hunk 2]
```diff
@@ -54,4 +58,4 @@ To remove a group by using the ObjectID, run `Remove-SPOGeoAdministrator -Object
 
 [Remove-SPOGeoAdministrator](https://docs.microsoft.com/powershell/module/sharepoint-online/remove-spogeoadministrator)
 
-[Set an alias (MailNickName) for a security group](https://docs.microsoft.com/en-us/powershell/module/azuread/set-azureadgroup)
\ No newline at end of file
+[Set an alias (MailNickName) for a security group](https://docs.microsoft.com/en-us/powershell/module/azuread/set-azureadgroup)
```
Comment (JohanFreelancer9) [inline Enterprise/add-a-sharepoint-geo-admin.md]:
```suggestion
[Set an alias (MailNickName) for a security group](https://docs.microsoft.com/powershell/module/azuread/set-azureadgroup)
```

File: Enterprise/add-a-sharepoint-geo-admin.md [hunk 3]
```diff
@@ -18,6 +18,10 @@ Some services - such as the term store - are administered from the central locat
 
 Global administrators and SharePoint Online administrators continue to have access to settings in the central location and all satellite locations.
 
+>[!Note]
```
Comment (nenonix) [inline Enterprise/add-a-sharepoint-geo-admin.md]:
```suggestion
> [!Note]
```

File: Enterprise/add-a-sharepoint-geo-admin.md (hunk 1 above)
Comment (nenonix) [inline Enterprise/add-a-sharepoint-geo-admin.md]:
```suggestion
> Geo admins can't manage Office 365 group-connected SharePoint sites but can add themselves as site collection administrators using the PowerShell cmdlet [Set-SPOUser](https://docs.microsoft.com/powershell/module/sharepoint-online/set-spouser?view=sharepoint-ps) .
```

File: Enterprise/add-a-sharepoint-geo-admin.md [hunk 4]
```diff
@@ -18,6 +18,10 @@ Some services - such as the term store - are administered from the central locat
...
+> Geo Admins cannot manage Office 365 group-connected SharePoint sites but can add themselves as Site Collection Admins to an Office 365 group-connected SharePoint sites using [Set-SPOUser](https://docs.microsoft.com/powershell/module/sharepoint-online/set-spouser?view=sharepoint-ps) PowerShell cmdlet.
+>
```
Comment (nenonix) [inline Enterprise/add-a-sharepoint-geo-admin.md]:
```suggestion
```

**Comment by mypil** [comment]
@get-itips  - please merge the commit on the copy/edit review done by JohanFreelancer9 / nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by MikePlumleyMSFT** [comment]
@get-itips please don't remove the title and description quotes in the metadata. They are required.

**Pull request closed without merging**